.travis.yml

```diff
@@ -8,45 +8,12 @@ services:
 env:
   matrix:
     - MODE=branchStrategy
-    - MODE=semantic PR_ONLY=true CHECK_NAME="Semantic Validator"
-    - MODE=model PR_ONLY=true CHECK_NAME="Model Validator"
-    - MODE=BreakingChange PR_ONLY=true CHECK_NAME="Breaking Changes"
-    - MODE=lintdiff PR_ONLY=true CHECK_NAME="Linter Diff" NODE_OPTIONS=--max-old-space-size=8192
 matrix:
   fast_finish: true
-  allow_failures:
-    - env: MODE=model PR_ONLY=true CHECK_NAME="Model Validator"
-    - env: MODE=BreakingChange PR_ONLY=true CHECK_NAME="Breaking Changes"
 install: true
 script:
   - >-
     if [[ $MODE == 'branchStrategy' ]]; then
```

Comment:
will you be removing this too in a following PR?

Reply:
We will keep `branchStrategy` for awhile and then remove travis-ci completely.